<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>continuous</class>
 <widget class="QDialog" name="continuous">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1247</width>
    <height>493</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QCAAxis" name="Axis1" native="true">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>280</width>
     <height>27</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>100</width>
     <height>10</height>
    </size>
   </property>
   <property name="description" stdset="0">
    <string>Start</string>
   </property>
   <property name="pvReadback" stdset="0">
    <string>u411pgm1:GetSweepStart</string>
   </property>
   <property name="units" stdset="0">
    <string>eV</string>
   </property>
   <property name="pv" stdset="0">
    <string>u411pgm1:SetSweepStart</string>
   </property>
  </widget>
  <widget class="QCAAxis" name="Axis2" native="true">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>60</y>
     <width>280</width>
     <height>27</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>100</width>
     <height>10</height>
    </size>
   </property>
   <property name="description" stdset="0">
    <string>End</string>
   </property>
   <property name="pvReadback" stdset="0">
    <string>u411pgm1:GetSweepEnd</string>
   </property>
   <property name="units" stdset="0">
    <string>eV</string>
   </property>
   <property name="pv" stdset="0">
    <string>u411pgm1:SetSweepEnd</string>
   </property>
  </widget>
  <widget class="QCAAxis" name="Axis3" native="true">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>60</y>
     <width>280</width>
     <height>27</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>100</width>
     <height>10</height>
    </size>
   </property>
   <property name="description" stdset="0">
    <string>Vel</string>
   </property>
   <property name="pvReadback" stdset="0">
    <string>u411pgm1:GetSweepVel</string>
   </property>
   <property name="units" stdset="0">
    <string>eV/s</string>
   </property>
   <property name="pv" stdset="0">
    <string>u411pgm1:SetSweepVel</string>
   </property>
  </widget>
  <widget class="QCAPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>1150</x>
     <y>40</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 0, 0);</string>
   </property>
   <property name="text">
    <string>STOP</string>
   </property>
   <property name="pv" stdset="0">
    <string>u411pgm1:MonoAktion.PROC</string>
   </property>
  </widget>
  <widget class="QCAPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>1150</x>
     <y>80</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>SweepInit</string>
   </property>
   <property name="pv" stdset="0">
    <string>u411pgm1:MonoAktion.PROC</string>
   </property>
  </widget>
  <widget class="QCAPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>1150</x>
     <y>120</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>SweepRun</string>
   </property>
   <property name="pv" stdset="0">
    <string>u411pgm1:MonoAktion.PROC</string>
   </property>
  </widget>
  <widget class="QCAAxis" name="Axis4" native="true">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>170</y>
     <width>280</width>
     <height>27</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>100</width>
     <height>10</height>
    </size>
   </property>
   <property name="description" stdset="0">
    <string>Startramp</string>
   </property>
   <property name="pvReadback" stdset="0">
    <string>u411pgm1:GetStartRamp</string>
   </property>
   <property name="units" stdset="0">
    <string/>
   </property>
   <property name="pv" stdset="0">
    <string>u411pgm1:GetStartRamp</string>
   </property>
  </widget>
  <widget class="QCAAxis" name="Axis5" native="true">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>170</y>
     <width>280</width>
     <height>27</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>100</width>
     <height>10</height>
    </size>
   </property>
   <property name="description" stdset="0">
    <string>State</string>
   </property>
   <property name="pvReadback" stdset="0">
    <string>u411pgm1:GetSweepState</string>
   </property>
   <property name="units" stdset="0">
    <string/>
   </property>
   <property name="pv" stdset="0">
    <string>u411pgm1:GetSweepState</string>
   </property>
  </widget>
  <widget class="QCAAxis" name="Axis6" native="true">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>170</y>
     <width>280</width>
     <height>27</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>100</width>
     <height>10</height>
    </size>
   </property>
   <property name="description" stdset="0">
    <string/>
   </property>
   <property name="pvReadback" stdset="0">
    <string>u411pgm1:monoGetEnergy</string>
   </property>
   <property name="units" stdset="0">
    <string>eV</string>
   </property>
   <property name="pv" stdset="0">
    <string>u411pgm1:monoGetEnergy</string>
   </property>
  </widget>
  <widget class="QCALabel" name="label">
   <property name="geometry">
    <rect>
     <x>970</x>
     <y>180</y>
     <width>59</width>
     <height>14</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 170, 127);</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="pv" stdset="0">
    <string>u411pgm1:Status</string>
   </property>
  </widget>
  <widget class="QCAAxis" name="Axis7" native="true">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>280</y>
     <width>280</width>
     <height>27</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>100</width>
     <height>10</height>
    </size>
   </property>
   <property name="description" stdset="0">
    <string>Vmin Faktor</string>
   </property>
   <property name="pvReadback" stdset="0">
    <string>u411pgm1:GetSweepVminFak</string>
   </property>
   <property name="units" stdset="0">
    <string/>
   </property>
   <property name="pv" stdset="0">
    <string>u411pgm1:SetSweepVminFak</string>
   </property>
  </widget>
  <widget class="QCAAxis" name="Axis8" native="true">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>280</y>
     <width>280</width>
     <height>27</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>100</width>
     <height>10</height>
    </size>
   </property>
   <property name="description" stdset="0">
    <string>Vmax Faktor</string>
   </property>
   <property name="pvReadback" stdset="0">
    <string>u411pgm1:GetSweepVmaxFak</string>
   </property>
   <property name="units" stdset="0">
    <string/>
   </property>
   <property name="pv" stdset="0">
    <string>u411pgm1:SetSweepVmaxFak</string>
   </property>
  </widget>
  <widget class="QCAAxis" name="Axis9" native="true">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>280</y>
     <width>280</width>
     <height>27</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>100</width>
     <height>10</height>
    </size>
   </property>
   <property name="description" stdset="0">
    <string>Offset</string>
   </property>
   <property name="pvReadback" stdset="0">
    <string>u411pgm1:GetSweepOffset</string>
   </property>
   <property name="units" stdset="0">
    <string>eV</string>
   </property>
   <property name="pv" stdset="0">
    <string>u411pgm1:SetSweepOffset</string>
   </property>
  </widget>
  <widget class="QCAAxis" name="Axis10" native="true">
   <property name="geometry">
    <rect>
     <x>950</x>
     <y>280</y>
     <width>280</width>
     <height>27</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>100</width>
     <height>10</height>
    </size>
   </property>
   <property name="description" stdset="0">
    <string>Tics</string>
   </property>
   <property name="pvReadback" stdset="0">
    <string>u411pgm1:SweepTics</string>
   </property>
   <property name="units" stdset="0">
    <string>eV</string>
   </property>
   <property name="pv" stdset="0">
    <string>u411pgm1:SetSweepTics</string>
   </property>
  </widget>
  <widget class="QCAAxis" name="Axis11" native="true">
   <property name="geometry">
    <rect>
     <x>950</x>
     <y>380</y>
     <width>280</width>
     <height>27</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>100</width>
     <height>10</height>
    </size>
   </property>
   <property name="description" stdset="0">
    <string>TGG</string>
   </property>
   <property name="pvReadback" stdset="0">
    <string>u411pgm1:TGG</string>
   </property>
   <property name="units" stdset="0">
    <string/>
   </property>
   <property name="pv" stdset="0">
    <string>u411pgm1:TGG</string>
   </property>
  </widget>
  <widget class="QCAComboBox" name="ComboBox_4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>390</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <property name="pv" stdset="0">
    <string>u411pgm1:SetSweepGapMeth</string>
   </property>
   <item>
    <property name="text">
     <string>quadr.</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>linear</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>370</y>
     <width>59</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>GapMeth</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>420</y>
     <width>59</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>quadr.</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCALabel</class>
   <extends>QLabel</extends>
   <header>qcalabel.h</header>
  </customwidget>
  <customwidget>
   <class>QCAAxis</class>
   <extends>QWidget</extends>
   <header>qcaaxis.h</header>
  </customwidget>
  <customwidget>
   <class>QCAPushButton</class>
   <extends>QPushButton</extends>
   <header>qcapushbutton.h</header>
  </customwidget>
  <customwidget>
   <class>QCAComboBox</class>
   <extends>QComboBox</extends>
   <header>qcacombobox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
